### DISCUSSION TITLE
Nextcloud OCC scan

### DISCUSSION THREAD

**Discussion by daschmidt1994** [alexbelgium/hassio-addons#644] [Q&A]
Is it possible in the Nextcloudaddon use occ commands for maintance or occ file scan?

**Comment by alexbelgium** [comment]
At launch yes, just put your script in /config/addons_autoscripts/nextcloud-ocr or something like that (exact path is described at Addon boot normally)

In theory my autoscript module is activated

**Comment by daschmidt1994** [comment]
thanks, that mean if the addon is running there is no way to use the occ comands e.g.

files:scan

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by alexbelgium** [comment]
Only way is through portainer (easy) or use the autoscript to set a cron job (complex)
